assets/js/components/Dashboard/PaMessaging/NewPaMessagePage/NewPaMessagePage.tsx

```diff
@@ -31,6 +40,8 @@ const NewPaMessagePage = () => {
   const [phoneticText, setPhoneticText] = useState("");
   const [audioPlaying, setAudioPlaying] = useState(false);
   const [audioReviewed, setAudioReviewed] = useState(false);
+  const [showErrorAlert, setShowErrorAlert] = useState(false);
+  const [errorMessage, setErrorMessage] = useState("");
```

Comment:
Do these need to be separate pieces of state? It seems like either we are showing an error _and_ there is an error message, or we are not showing an error (and there is no error message).

Reply:
That's a good point. The boolean can just be `.length > 0`.